js/languages/en-US.json

```diff
@@ -85,6 +85,41 @@
     "generalTab": "General",
     "pageTab": "Page",
     "addressesTab": "Addresses",
+    "advancedTab" : {
+      "_sectionName" : "Advanced",
+      "smtp" : {
+        "_sectionName" : "SMTP Integration",
```

Comment:
The _ is to distinguish between words to do with model slots ( names of slots, errors about slots ), and words to do with things which are in the view and are meta to the model slots. Things like actions, or section headings.